### ISSUE TITLE
username requires manual verification on instagram (429).

### ISSUE THREAD

**Issue by dremara** [crock/cli-checker#2]
excuse me if this is an easy solve, i'm not a tech buff.
i'm running 100 threads on a list of 70000 words to check for instagram using approx 350 free proxies that all check out as good.

however 95% of my returns are 'requires manual verification on instagram (429).' and there will then be a very brief period where the script works for a few usernames before going back to the 429 error.

i know i'm doing something wrong, i know the answer may be obvious to most, but i'd really appreciate some help with this one.
thanks.
andreas

**Comment by dremara** [comment]
oh, also the available names are not outputting properly to available.txt, all lines output read

https://www.instagram.com/accounts/web_create_ajax/attempt/

**Comment by yellow** [comment]
Can confirm.
But I faced this issue while checking twitters.
I even tried using a single thread but the issue is not solved.

**Comment by crock** [comment]
@dremara @yellow Yes, I am aware of the issue and am working on creating a caching system to solve this. The issue is due to Instagram's strict rate limits. You can only make so many requests per 15 minute window.

A quick and dirty workaround is to not use more than 2 threads and 300 word lists when checking Instagram or Twitter. All the other services shouldn't have a problem with the 100 threads.

**Comment by ZXT-Tech** [comment]
@crock the problem is the proxy checker and the whole proxy integration isn't working.

scripts@L2:~/cli-checker$` python3 OGCheckr.py
Checking and filtering out bad proxies...
Starting up threads...
[Thread-0] has started.
notworking is working...
github.com:8080 is working...
something:3128 is working...
 is working...
Done.

Searched 4 proxies and filtered out 0 bad proxies in 0.44285035133361816 seconds
Starting up threads...

**Comment by crock** [comment]
the proxy checker looks like it is working based on that output you pasted?

**Comment by lgg** [comment]
@crock i have same issue with github and 4 threads.

**Comment by crock** [comment]
>  i have same issue with github and 4 threads.
@lgg 

I think GitHub heavily rate limits. You can try using some free proxy lists (like 100 or so) or get a couple of premium "rotating" proxies.
